katsdpcontroller/scheduler.py

```diff
@@ -797,9 +797,7 @@ def __repr__(self):
 
 
 class InterfaceRequest(ResourceRequestsContainer):
-    """Request for resources on a network interface. At the moment only
-    a logical network name can be specified, but this may be augmented in
-    future to allocate bandwidth.
+    """Request for resources on a network interface.
```

Comment:
This change isn't actually related to the PR, but I noticed this comment was very out of date.